charts/sf-datapath/sub_charts/autoscaling/templates/cp-kafka-rest-hpa.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Capabilities.APIVersions.Has "autoscaling/v2beta2" }}
 {{- if .Values.cpKafkaRestProxy.enabled -}}
```

Comment:
Please move this to top of file so that single if condition will suffice
It should not be replicated